README.md

```diff
@@ -43,7 +60,80 @@ VisuMorph includes a variety of functions for image manipulation:
 - **Hue Change(`change_hue`)**: Adjust the hue of images, allowing for color shifting and mood setting in visuals.
 - **Scaling(`scale`)**: Resize images, either uniformly or non-uniformly, without losing the essence of the visual content.
 
-## Run the Tests
+## Quick Start Examples
+
+For a quick glimpse into what VisuMorph can do, here are a few simple examples:
+
+### Flip an Image
+Flip an image horizontally or vertically. This example flips an image horizontally.
+
+```python
+import visumorph as vm
+
+# Load your image
+img = vm.load_image("path/to/image.jpg")
+
+# Flip the image horizontally
+flipped_img = vm.flip(img, direction='horizontal')
+
+# Save the flipped image
+flipped_img.save("path/to/flipped_image.jpg")
+```
+
+### Rotate an Image
+Rotate an image by a specified angle in degrees. This example rotates an image by 90 degrees.
+
+```python
+import visumorph as vm
+
+# Load your image
+img = vm.load_image("path/to/image.jpg")
+
+# Rotate the image by 90 degrees
+rotated_img = vm.rotate(img, rotation=90)
+
+# Save the rotated image
+rotated_img.save("path/to/rotated_image.jpg")
+```
+
+### Change Image Hue
+Change the hue of an image. This example changes the hue by a delta value.
+
+```python
+import visumorph as vm
+
+# Load your image
+img = vm.load_image("path/to/image.jpg")
+
+# Change the hue of the image
+hue_changed_img = vm.change_hue(img, delta_hue=30)
```

Comment:
Are you sure delta_hue would be ok when outside of the range [0, 1]? This is not what the doctoring says.

Reply:
Resolving this as the line is rectified in commit [2ed1c87](https://github.com/UBC-MDS/VisuMorph/pull/93/commits/2ed1c87b884d3a5ef790013a57d4177bff60cf17).